app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/actions.ts

```diff
@@ -310,3 +315,13 @@ export const getSubmissionsByFormat = async ({
     }
   }
 };
+
+export const confirmSubmissionCodes = async (confirmationCodes: string[], formId: string) => {
+  try {
+    const { ability } = await authCheckAndRedirect();
+
+    return confirmResponses(ability, confirmationCodes, formId);
+  } catch (e) {
+    throw new Error("There was an error. Please try again later.");
```

Comment:
should we pass in the e object?

Reply:
I purposefully left it out since this traverses the server / client boundary to avoid leaking any sensitive information back to the client.

Reply:
make sense, was hoping we could log the actual error somewhere to help troubleshoot, perhaps Im missing a layer

Reply:
I get what you're saying.  I'll go back through the layers to see if there is a chance that any unhandled errors could bubble up. The try / catch at that level what just a pattern we started adopting for the boundary actions that didn't have any precise error return objects.

Reply:
@wmoussa-gc I've added a catch all error that is logged server side as a Warning.